django/db/migrations/executor.py

```diff
@@ -79,7 +82,8 @@ def migrate(self, targets, plan=None, fake=False, fake_initial=False):
         all_backwards = all(backwards for mig, backwards in plan)
 
         if not plan:
-            pass  # Nothing to do for an empty plan
+            # Nothing to do for an empty plan
```

Comment:
Maybe extend the comment by something like "... empty plan, except for building the post migrate project state" -- "nothing to do" and "doing something" don't go well together